File: api/examples/vb.net/STATE.vb
<Serializable()>
Public Class STATE
    Public License As Integer = 0
    Public Drivers As Integer = 0
    Public Browsers As Integer = 0
    Public Options As Integer = 0
    Public InitErrors As New Dictionary(Of Integer, String)
    Public Update As Integer = 0
    Public UpdateVersion As String = String.Empty
    Public UpdateMessage As String = String.Empty
    Public SavedBubbles As Integer = 0
    Public RunningBubbles As Integer = 0
End Class

File: api/examples/c#/STATE.cs
using System;
using System.Collections.Generic;

[Serializable()]
public partial class STATE
{
    public int License = 0;
    public int Drivers = 0;
    public int Browsers = 0;
    public int Options = 0;
    public Dictionary<int, string> InitErrors = new Dictionary<int, string>();
    public int Update = 0;
    public string UpdateVersion = string.Empty;
    public string UpdateMessage = string.Empty;
    public int SavedBubbles = 0;
    public int RunningBubbles = 0;
}